eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -49,6 +48,8 @@ function Get-ChangeLogEntriesFromContent {
     $initialAtxHeader = $matches["HeaderLevel"]
   }
 
+  $SECTIONS_HEADER_REGEX = "^${initialAtxHeader}##\s(?<sectionName>.*)"
```

Comment:
Isn't this used outside of this function in the remove-empty sections as well? If so moving it in here will not allow you to use it that function as well.

Reply:
I have added $sectionHeaderRegex to both functions.